web/pdf_find_controller.js

```diff
@@ -772,7 +772,7 @@ class PDFFindController {
 
   #calculateMatch(pageIndex) {
     const query = this.#query;
```

Comment:
Oh, I think we need the following instead since I previously missed the fact that [this getter](https://github.com/mozilla/pdf.js/blob/de2a44a5588604561d2e9ebf7825063a9d6f31b1/web/pdf_find_controller.js#L611-L626) also access the state.
```suggestion
    if (!this.#state) {
      return
    }
    const query = this.#query;
```